project_task_category/models/project_task_category.py

```diff
@@ -11,4 +11,5 @@ class ProjectTask(models.Model):
     categ_id = fields.Many2one(
         'project.category',
         'Category',
+        domain=[('code', '=', 'categ_id')]
```

Comment:
@dreispt I get your point of view but I'd rather have this filter domain and later if you want could be proposed as well.

Just for curiosity how could you categorize or distinguish between project domains? Seems very complex without another "Category level" just for projects

cc @elicoidal